src/app/join-us/page.tsx

```diff
@@ -1,32 +1,166 @@
-import { Typography } from '@mui/material';
+import CoreValues from '@/components/core-values/core-values';
+import Faq from '@/components/faq/faq';
+import { Box, Divider, Paper, Typography } from '@mui/material';
 import Container from '@mui/material/Container';
+import Image from 'next/image';
 import React from 'react';
 import './join-us.scss';
 
 export default function JoinUs() {
   return (
     <Container maxWidth="lg" className="container">
-      <Typography className="join-us-header" variant="h3">
-        Join Us
-      </Typography>
-      <Typography variant="body2" paragraph>
-        Unlock the power of your skills and expertise to drive positive change
-        in British Columbia through OpenBC. At OpenBC, we&apos;re passionate
-        about leveraging your unique abilities for the advancement of our
-        province. Whether you&apos;re a tech enthusiast, a creative visionary,
-        or a community leader, your skills can play a crucial role in addressing
-        challenges and creating opportunities.
-      </Typography>
-      <Typography variant="body2" paragraph>
-        Ready to contribute to British Columbia through Open source projects?
-        Join us at OpenBC and become a part of our active and welcoming
-        community. Here&apos;s how you can engage:
-      </Typography>
-      <Typography variant="body2">
-        <strong>
-          Email openbc.ca@gmail.com for opportunities to contribute.
-        </strong>
+      <Box className="sub-container-2-col">
+        <Box sx={{ width: '50%' }}>
+          <Image
+            src="https://source.unsplash.com/random?wallpapers"
+            alt="Description"
+            width={500}
+            height={500}
+            style={{ width: '100%' }}
+          />
+        </Box>
+        <Box sx={{ width: '50%', marginX: '2em' }}>
+          <Typography className="sub-header" variant="h5">
+            Our Mission
+          </Typography>
+          <Typography variant="body2" paragraph>
+            At OpenBC, we&apos;re passionate about leveraging your unique
+            abilities for the advancement of our province.
+            <strong>
+              Contribute to Open Source Projects for BC, from Anywhere, with
+              OpenBC.
+            </strong>
+          </Typography>
+        </Box>
+      </Box>
+      <Box sx={{ marginY: '4em' }}>
+        <Typography className="sub-header" variant="h5">
+          Visions
+        </Typography>
+        <Typography variant="body2" paragraph>
+          Quis neque, eu et ipsum amet, vel et. Varius integer enim pellentesque
+          ornare pharetra faucibs arcu. Mauris blandit egestas nibh. eu et ipsum
+          amet, vel et. Varius integer enim pellentesque ornare pharetra faucibs
+          arcu. Mauris blandit egestas nibh.
+        </Typography>
+      </Box>
+      <Box sx={{ width: '40%', alignSelf: 'center' }}>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
```suggestion
        <Box sx={{ display: 'flex', justifyContent: 'space-around' }}>
```

This is closer to the Figma design:

## Design

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/e0e8b012-4ee2-4f90-b7e1-2f364e328d42)

## Before

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/c61041fb-db95-44c7-9661-915fc52bd2ad)

## After

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/333b43e9-41fa-48b9-9a20-d5751a4d0721)

Note the padding on the outer edges of the element